platform_api/orchestrator/jobs_service.py

```diff
@@ -61,6 +62,12 @@ def __init__(self, user: str) -> None:
         super().__init__(f"No credits left for user '{user}'")
 
 
+@dataclass(frozen=True)
+class UserClusterConfig:
+    config: ClusterConfig
+    orgs: List[Optional[str]]
```

Comment:
curious why `Optional` here?

Reply:
I think it is either `[None, ..]` or `["org1", "org2"]`. seems like we could simplify, or at least put a comment.

Reply:
Let's put only comment here. If this place shows up to be problematic, we can split it into `orgs: List[str] + direct_access: bool`.

Reply:
thanks